src/vision/how_to_vision/comment.md

```diff
@@ -39,4 +39,4 @@ Here are some examples of good questions for "shiny future" stories:
 * How does NAME do X in this future?
 * It seems like this would interfere with X, which is important for application A. How would NAME handle that case in this future?
 
-You should not be afraid to raise technical concerns -- we need to have a robust technical discussion! But do so in a way that leaves room to find an answer that satisfies both of you.
\ No newline at end of file
+You should not be afraid to raise technical concerns -- we need to have a robust technical discussion! But do so in a way that leaves room to find an answer that satisfies both of you.
```

Comment:
I think GitHub must have inserted a newline here from the web UI. Not sure what else might've changed here.